File: OPENAPI-REST-API/swagger-client/csharp-dotnet2/src/main/CsharpDotNet2/myadmin-client-csharp-dotnet2/Model/MailAttachment.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// (optional) File attachments to include in the email.  The file contents must be base64
  /// </summary>
  [DataContract]
  public class MailAttachment {
    /// <summary>
    /// The filename of the attached file.
    /// </summary>
    /// <value>The filename of the attached file.</value>
    [DataMember(Name="filename", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "filename")]
    public string Filename { get; set; }

    /// <summary>
    /// The file contents base64 encoded
    /// </summary>
    /// <value>The file contents base64 encoded</value>
    [DataMember(Name="data", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "data")]
    public string Data { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class MailAttachment {\n");
      sb.Append("  Filename: ").Append(Filename).Append("\n");
      sb.Append("  Data: ").Append(Data).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: OPENAPI-REST-API/swagger-client/dart/lib/model/mail_attachment.dart
part of swagger.api;

class MailAttachment {
  /* The filename of the attached file. */
  String filename = null;
/* The file contents base64 encoded */
  String data = null;

  MailAttachment();

  @override
  String toString() {
    return 'MailAttachment[filename=$filename, data=$data, ]';
  }

  MailAttachment.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    filename = json['filename'];
    data = json['data'];
  }

  Map<String, dynamic> toJson() {
    return {
      'filename': filename,
      'data': data
     };
  }

  static List<MailAttachment> listFromJson(List<dynamic> json) {
    return json == null ? new List<MailAttachment>() : json.map((value) => new MailAttachment.fromJson(value)).toList();
  }

  static Map<String, MailAttachment> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, MailAttachment>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new MailAttachment.fromJson(value));
    }
    return map;
  }
}
